cli/command/service/logs.go

```diff
@@ -49,13 +49,14 @@ func newLogsCommand(dockerCli *command.DockerCli) *cobra.Command {
 			opts.target = args[0]
 			return runLogs(dockerCli, &opts)
 		},
-		Tags: map[string]string{"experimental": ""},
```

Comment:
And by unexpected behavior, I might flags might not work or might be ignored, but there should not be panics.